### PR TITLE
Sed 1059 tables artefactselecti

### PR DESCRIPTION
**Pull request by chillbirdo**

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/plan/components/plan-artefact-list/plan-artefact-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  selector: 'step-plan-artefact-list',
+  templateUrl: './plan-artefact-list.component.html',
+  styleUrls: ['./plan-artefact-list.component.scss'],
+})
+export class PlanArtefactListComponent {
+  @Output() onSelection = new EventEmitter<string>();
+
+  constructor(
+    readonly _artefactService: ArtefactService,
+    readonly _controllerApiService: AugmentedControllerService,
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/plan/components/plan-artefact-list/plan-artefact-list.component.ts:18]: Can not find the usage of this service or it's datasource.
Is it required? If not I think both field and augmented service, can be removed

**Review by dvladir — CHANGES_REQUESTED** [review]
Seems `augmented-controller-service` isn't necessary. Can not find it's datasource usage.
If it so, I guess it can be removed

**Comment by chillbirdo** [comment]
correct, I forgot to clean this up. I also removed all `public` modifiers ;)

**Review by dvladir — DISMISSED** [review]

**Comment by neogucky** [comment]
@chillbirdo can you do the merge? It looks like there are differences between the implementation here and in  "Sed 1054 tables functionselecti"

**Review by neogucky — DISMISSED** [review]

**Comment by chillbirdo** [comment]
done, ready to be merged!

**Review by neogucky — APPROVED** [review]
looks good!

**Pull request merged**